Sources/VaporToolbox/Clean.swift

```diff
@@ -22,7 +23,7 @@ public final class Clean: Command {
         let cleanBar = console.loadingBar(title: "Cleaning")
```

Comment:
We should make a warning before this happens that `vapor update` which calls `swift package update` is better